features/form.yml.dist

```diff
@@ -49,8 +49,8 @@ compat_features:
   #   edge: "12"
   #   firefox: "1"
   #   firefox_android: "4"
-  #   safari: ≤4
-  #   safari_ios: ≤3.2
+  #   safari: "3"
+  #   safari_ios: "2"
```

Comment:
Not all the way fixed. I'll reopen https://github.com/mdn/browser-compat-data/issues/24135 after merging